python_modules/dagster-airflow/dagster_airflow_tests/test_project/demo_pipeline_static__scaffold.py

```diff
@@ -0,0 +1,168 @@
+'''Static scaffolding autogenerated by dagster-airflow from pipeline demo_pipeline with config:
+
+    context:
+      default:
+        config: {log_level: DEBUG}
+    solids:
+      multiply_the_word:
+        config: {factor: 2}
+        inputs:
+          word: {value: bar}
+    
+
+By convention, users should attempt to isolate post-codegen changes and customizations to the
+"editable" demo_pipeline_editable__scaffold.py file, rather than changing the definitions in this
+"static" file. Please let us know if you are encountering use cases where it is necessary to make
+changes to the static file.
+'''
+
+from airflow import DAG
+from airflow.operators.dagster_plugin import DagsterOperator
+
+
+CONFIG = '''
+    {
+      context: {
+        default: {
+          config: {
+            log_level: "DEBUG"
+          }
+        }
+      },
+      solids: {
+        multiply_the_word: {
+          config: {
+            factor: 2
+          },
+          inputs: {
+            word: {
+              value: "bar"
+            }
+          }
+        }
+      }
+    }
+'''.strip('\n').strip(' ')
+
+PIPELINE_NAME = 'demo_pipeline'
+
+STEP_EXECUTIONS_MULTIPLY__THE__WORD_WORD_INPUT__THUNK = '''
+[
+  {
+    stepKey: "multiply_the_word.word.input_thunk"
+    marshalledInputs: 
+    [
+    ],
+    marshalledOutputs: 
+    [
+      {
+        outputName: "input_thunk_output",
+        key: "/tmp/results/multiply__the__word_word_input__thunk___input__thunk__output.pickle"
+      }
+    ],
+  }
+]
+'''.strip('\n')
+
+STEP_EXECUTIONS_MULTIPLY__THE__WORD_TRANSFORM = '''
+[
+  {
+    stepKey: "multiply_the_word.transform"
+    marshalledInputs: 
+    [
+      {
+        inputName: "word",
+        key: "/tmp/results/multiply__the__word_word_input__thunk___input__thunk__output.pickle"
+      }
+    ],
+    marshalledOutputs: 
+    [
+      {
+        outputName: "result",
+        key: "/tmp/results/multiply__the__word_transform___result.pickle"
+      }
+    ],
+  }
+]
+'''.strip('\n')
+
+STEP_EXECUTIONS_COUNT__LETTERS_TRANSFORM = '''
```

Comment:
This seems a bit dodgy. We should be making these results scoped by run_id at some level. Not going to block merging on this but we should create an issue to fix this